up-l2/dispatchers/README.adoc

```diff
@@ -30,95 +30,88 @@ SPDX-License-Identifier: Apache-2.0
 
 NOTE: These communication layer requirements are still for point-2-point uE communication to and from a dispatcher
 
-* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the CE to the intended Receiver
-  ** *MUST* queue CEs not successfully acknowledged (transport level at-least-once delivery confirmation described above)
-  ** *MUST* attempt to retry transmission of the CE. Retry policy is specific to the dispatcher implementation
-  ** Dispatcher *MUST NOT* discard CEs unless either CE has expired (CE.ttl), or the egress queue is full. CEs that cannot be delivered are sent to a Dead Letter Office Topic
+* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the message to the intended receiver
+  ** *MUST* queue messages not successfully acknowledged (transport level at-least-once delivery confirmation described above)
+  ** *MUST* attempt to retry transmission of the message. Retry policy is specific to the dispatcher implementation
+  ** Dispatcher *MUST NOT* discard message unless either the message has expired (see https://github.com/eclipse-uprotocol/uprotocol-core-api/blob/main/src/main/proto/uattributes.proto[`UAttribute:ttl`] for more details), or the egress queue is full. Messages that cannot be delivered are sent to a Dead Letter Office Topic (DLT)
+
+NOTE: Implementation of the DLT is uPlatform specific and is not covered in this document 
 
-* *MAY* support additional CE delivery policies in general or per topic in the future
+* *MAY* support additional message delivery policies in general or per topic in the future
 * *SHOULD* dispatch in order that it received the CE
 * *MAY* batch CEs when delivering to the Receiver
 * CEs that cannot be delivered *MUST* be sent to the Dead Letter topic (DLT)
-  ** DLT *MUST* include at least the CE header, SHOULD contain the full CE
-  ** DLT *MUST* include the reason for the failed delivery attempt using  error codes defined in google.rpc.Code
-  ** uEs MUST be able to subscribe to the DLT to be notified of message deliver failures
+  ** DLT *MUST* include at least the message header, *SHOULD* contain the full Message
+  ** DLT *MUST* include the reason for the failed delivery attempt using per https://github.com/eclipse-uprotocol/uprotocol-core-api/blob/main/src/main/proto/ustatus.proto[UStatus] error codes
+  ** uEs *SHOULD* be able to subscribe to the DLT to be notified of message deliver failures
 
 If the uP-L1 delivery method is push:
 
-* *SHALL* provide an API to start/stop dispatching of CEs per-topic, this is to avoid having to queue CEs on the Receiver if the Receiver is not ready to receive the CEs
+* *SHOULD* provide an API to start/stop dispatching of messages per-topic, this is to avoid having to queue messages on the Receiver if the Receiver is not ready to receive
```

Comment:
This is coupling the sender and receiver.
In a distributed system, uEs are separated by space and time - they are not directly connected.
This is the whole idea. Publishers send and consumers consumer.
A publisher might have more than one consumer - one is happily consuming while another is shut down and will come back up later and continue consuming from where it left off.
Does this mean that the publisher will stop because one of the consumers is having a difficult day?